### PR TITLE
8 implement travis ci

### PR DESCRIPTION
**Pull request by redmyt**

### CODE REVIEW

File: eventually/manage.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# pylint: disable=missing-docstring
```
Comment (lhalam) [inline eventually/manage.py:2]: unless Is this file not ignored in the file _.pylintrc_?
`+ignore=CVS, migrations, apps.py, urls.py, wsgi.py, settings.py, local_settings.py, manage.py`

**Review by lhalam — APPROVED** [review]

File: eventually/manage.py (hunk 1 above)
Comment (redmyt) [reply eventually/manage.py:2]: despite "manage.py" is have to be ignored, the command `pylint --rcfile=.pylintrc *` verify it and returns that two errors that I've explicitly commented.

**Review by lhalam — APPROVED** [review]
ok, can you merge.

File: eventually/manage.py (hunk 1 above)
Comment (lhalam) [reply eventually/manage.py:2]: ok, can you merge.

**Pull request merged**